### ISSUE TITLE
Ignores `@inheritDoc` declarations

### ISSUE THREAD

**Issue by mydea** [gund/eslint-plugin-deprecation#80]
I noticed this more or less accidentally - if you e.g. have code like this:

```ts
interface MyInterface {
  /**
   * This is old!
   * @deprecated Do not use this anymore.
   */
   oldProp?: string;
}

class MyClass implements MyInterface {
  /** @inheritDoc /*
  oldProp?: string;
}

const obj = new MyClass();
obj.oldProp = 'xx'; // does not raise an eslint error :(
```

Honestly I am not sure how easy it is to fix this, but got bitten by this quite a bit because I expected this to work 😅

**Comment by gund** [comment]
Hey, did you try to remove the jsdoc from the class altogether and see if it works without `@inheritDoc`?
